modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/package.scala

```diff
@@ -31,7 +31,9 @@ package object server {
   }
 
   implicit class SeqActionOps[A](a: SeqAction[A]) {
-    def runSeqAction: TrySeq[A] = a.run.unsafePerformSyncAttempt.leftMap[SeqexecFailure](SeqexecException).join
+    def runSeqAction: TrySeq[A] = a.run.handle{
+      case e: Exception => SeqexecException(e).left
+    }.unsafePerformSync
```

Comment:
I'm not sure where `runSeqAction` is being called. Perhaps we can remove it if we are not using it anywhere.